analyzer/main.zeek

```diff
@@ -289,7 +289,7 @@ function set_session(c: connection, message_id: int, opcode: LDAP::ProtocolOpcod
 @if (Version::at_least("4.2.0"))
 event analyzer_confirmation(c: connection, atype: Analyzer::Tag, aid: count) {
```

Comment:
Actually I think *this* should be `AllAnalyzers::Tag`.

Reply:
I should really be the other way around, https://github.com/zeek/zeek/blob/bf7b6e5ed09906e0f3345eb9b0c677f855c908a2/src/event.bif#L382-L383=.

Reply:
No, as with e.g., zeek-4.2.2 that would trigger

```
use of undeclared alternate prototype (event(c:connection; atype:error; aid:count;) and analyzer_confirmation
```

Reply:
I'm not entirely sure why, but I changed that line to be `AllAnalyzers::Tag` and it worked just fine here. The plugin's tests passed and it installed. I didn't attempt to use it, but assuming the tests covered everything I don't know why it didn't work.

That said, I do think that `event.bif` should be changed to match the correct type, even if everywhere that uses that event seems to work.

Reply:
I am not sure why `AllAnalyzers::Tag` didn't work for me locally, now it does. Sorry for the back and forth.